src/components/app/MenuButtons.js

```diff
@@ -589,6 +604,69 @@ const MenuButtons = ({
   predictUrl,
 }: MenuButtonsProps) => (
   <div className="menuButtons">
+    <ButtonWithPanel
+      className="menuButtonsOpenMetainfo"
+      label="Profile info"
+      panel={
+        <ArrowPanel className="arrowPanelOpenMetainfo">
+          <h2 className="arrowPanelSubTitle">Timing</h2>
+          <div className="arrowPanelSection">
+            <span>
+              <span className="metainfolabel">startTime: </span>
+              {_formatDate(profile.meta.startTime)}
+            </span>
+            <br />
+            <span>
+              <span className="metainfolabel">Interval:</span>{' '}
```

Comment:
Do we need the trailing spaces here?

Reply:
I'm in favor of those, yes.

Reply:
Can you explain what they are for? It's non-obvious to me. It might be good to leave a small comment in the code as well.